modules/default-values/assets/js/editor/commands/base.js

```diff
@@ -63,4 +63,12 @@ export default class Base extends $e.modules.CommandBase {
 		// TODO: Maybe in the command?
 		elementor.kitManager.renderGlobalsDefaultCSS();
 	}
+
+	showLoader() {
+		jQuery( '#elementor-preview-loading' ).show();
+	}
+
+	hideLoader() {
+		jQuery( '#elementor-preview-loading' ).fadeOut( 600 );
+	}
```

Comment:
For `600` use `const` 

Suggestion: `hookUI` or `$e.uiStates`.

Reply:
I put this logic in separate methods to allow us to change it later, I just copy and paste from another place that has loading, not sure the change of the loaders is in the scope of the current task.